app/scripts/metamask-controller.js

```diff
@@ -6638,6 +6680,21 @@ export default class MetamaskController extends EventEmitter {
     this._notifyChainChange();
   }
 
+  /**
+   * Execute side effects of a removed account.
+   *
+   * @param {string} address - The address of the account to remove.
+   * @returns {Promise<void>}
+   */
+  async _onAccountRemoved(address) {
```

Comment:
Nit: this seems to me it can be just a void can we remove the async keyword ? and await when calling the function

Reply:
good point! done in https://github.com/MetaMask/metamask-extension/pull/27025/commits/d84e1671c72d5d09d90d82e4147396ddd4c681ab